cms/static/js/certificates/views/certificate_details.js

```diff
@@ -0,0 +1,71 @@
+// Backbone Application View: Certificate Details
+
+define([
+    'js/views/baseview', 'underscore', 'js/certificates/models/signatory', 'js/certificates/views/signatory_details', 'gettext', 'underscore.string'
+],
+function(BaseView, _, Signatory, SignatoryDetails, gettext, str) {
+    'use strict';
+    var CertificateDetailsView = BaseView.extend({
+        tagName: 'div',
+        events: {
+            'click .edit': 'editCertificate',
+            'click .show-details': 'showDetails',
+            'click .hide-details': 'hideDetails'
+        },
+
+        className: function () {
+            // Determine the CSS class names for this model instance
+            var index = this.model.collection.indexOf(this.model);
+
+            return [
+                'collection',
+                'certificates',
+                'certificate-details',
+                'certificate-details-' + index
```

Comment:
Not sure of the need for this indexy class name... I may be mistaken, but there does not appear to be any styles / js code using it. My first reaction was that there are style-rules that could let you change the style according to position-index in a list, instead of appending an index to the className.